components/operator/KeystoreForm.tsx

```diff
@@ -124,8 +125,8 @@ export const KeystoreForm = ({
       const clusterParams = {
         contractAddress: contractAddress,
         nodeUrl: "https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161",
-        // ownerAddress: walletAddress as `0x${string}`,
-        ownerAddress: "0x35E928fBAd7a404fbcffA51c85d2ccFd045663CB",
+        ownerAddress: walletAddress as `0x${string}`,
```

Comment:
I think this type cast is not necessary.

In any case, better use the `Address` type from wagmi